indico/modules/events/editing/client/js/editing/timeline/FileManager/util.js

```diff
@@ -108,11 +127,14 @@ export async function deleteFile(uuid) {
   }
 }
 
-export function mapFileTypes(fileTypes, files) {
+export function mapFileTypes(fileTypes, files, uploadableFiles = []) {
   return fileTypes.map(fileType => ({
     ...fileType,
     files: files.filter(file => file.fileType === fileType.id).map(f => ({...f, claimed: true})),
     invalidFiles: [],
+    uploadableFiles: uploadableFiles.filter(file =>
+      fileType.extensions.includes(file.filename.split('.').pop())
```

Comment:
I think you are not handling the case where a file type has no extension restrictions

Reply:
I’ll add an exception. Thanks!

Reply:
PS: I added a condition inside the `filter` instead of an outside check because I just thought it would be a good idea to preserve the immutability `filter` provides.